### PR TITLE
Ide 3972 16

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

**Comment by yanan** [comment]
it retest

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/gamerson/liferay-ide/pull/4363. [See changes here.](https://github.com/gamerson/liferay-ide/compare/eb56afffa~2...eb56afffa) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/ui/upgrade/UpdateWorkspacePluginVersionCommand.java [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+	@Reference
+	private ResourceSelection _resourceSelection;
+
+	@Reference
+	private UpgradeCompare _upgradeCompare;
+
+	@Reference
+	private UpgradePlanner _upgradePlanner;
+
+	private final String _workspacePluginLatestVersion = "2.0.0";
```
[the remark this answers is not in the document]
Comment (gamerson) [reply tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/ui/upgrade/UpdateWorkspacePluginVersionCommand.java]: @simonjhy  let me know if you get these comments.  Thanks

[the remark this answers is not in the document]
Comment (simonjhy) [reply tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/ui/upgrade/UpdateWorkspacePluginVersionCommand.java]: ok

File: tools/plugins/com.liferay.ide.upgrade.plan.core/src/com/liferay/ide/upgrade/plan/core/internal/07-liferay-code-upgrade-plan.markdown [hunk 2]
```diff
@@ -67,11 +67,11 @@ Here are the steps for upgrading your development environment and custom plugins
             <div class="title">Update Repository URL</div>
             <div class="description">Update your repository URL.</div>
         </li>
-        <li>
+        <li commandId="update_workspace_plguin_version">
```
[the remark this answers is not in the document]
Comment (simonjhy) [reply tools/plugins/com.liferay.ide.upgrade.plan.core/src/com/liferay/ide/upgrade/plan/core/internal/07-liferay-code-upgrade-plan.markdown]: ok

File: tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/ui/upgrade/UpdateWorkspacePluginVersionCommand.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (simonjhy) [reply tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/ui/upgrade/UpdateWorkspacePluginVersionCommand.java]: Hi Greg, I created new issue(https://github.com/gamerson/liferay-ide/issues/4365).

**Pull request closed without merging**